packs/feat-effects/effect-assisting-shot.json

```diff
@@ -4,28 +4,81 @@
-            "value": -1
+            "unit": "rounds",
+            "value": 1
         },
         "level": {
             "value": 2
         },
         "publication": {
-            "license": "OGL",
-            "remaster": false,
-            "title": "Pathfinder Core Rulebook"
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Player Core"
         },
         "rules": [
             {
-                "key": "FlatModifier",
-                "selector": "attack",
-                "type": "circumstance",
-                "value": 1
+                "actorFlag": true,
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Attack.success",
+                        "predicate": [
+                            {
+                                "or": [
+                                    "parent:context:check:outcome:2",
+                                    {
+                                        "not": {
+                                            "gte": [
+                                                "parent:context:check:outcome",
+                                                0
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": 1
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Attack.criticalSuccess",
+                        "predicate": [
+                            {
+                                "or": [
+                                    "parent:context:check:outcome:3",
+                                    {
+                                        "not": {
+                                            "gte": [
+                                                "parent:context:check:outcome",
+                                                0
+                                            ]
+                                        }
+                                    }
+                                ]
+                            }
+                        ],
+                        "value": 2
+                    }
+                ],
+                "flag": "assistingShotBonus",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess"
+            },
+            {
+                "affects": "origin",
+                "key": "EphemeralEffect",
+                "predicate": [
+                    {
+                        "not": "origin:signature:{item|origin.signature}"
+                    }
+                ],
+                "selectors": [
+                    "attack"
+                ],
+                "uuid": "Compendium.pf2e.feat-effects.Item.Effect: Assisting Shot Bonus"
```

Comment:
Why use an ephemeral here?

I feel like this should just be a single effect that goes on the next attacker. As is this is not clear from the description that the effect should go on the target.